### ISSUE TITLE
Add custom labels

### ISSUE THREAD

**Issue by tehleet** [kubernetes/autoscaler#3633]
https://github.com/kubernetes/autoscaler/blob/d054bb248f9689eef39be21f796fafd236437073/charts/cluster-autoscaler-chart/templates/_helpers.tpl#L42-L49

at the moment labels helper doesn't allow to add "custom" labels, would be great to enable this, etc

```
# templates/_helpers.tpl
{{/*
Return labels, including instance and name.
*/}}
{{- define "cluster-autoscaler.labels" -}}
{{ include "cluster-autoscaler.instance-name" . }}
app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
helm.sh/chart: {{ include "cluster-autoscaler.chart" . | quote }}
{{- if .Values.labels }}
{{- range $k, $v := .Values.labels }}
{{ $k }}: {{ $v | quote }}
{{- end -}}
{{- end -}}
{{- end -}}
```

and following change to values.yaml file
```
## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
# affinity -- Affinity for pod assignment
affinity: {}

# Aditinional custom labels for resources created by this helm release
labels: {}
  # ci_build_no: 1
  # git_hash: b23as42d

autoDiscovery:
  # cloudProviders `aws`, `gce` and `magnum` are supported by auto-discovery at this time
...
```

**Comment by mgarstecki** [comment]
I've submitted a PR for that: https://github.com/kubernetes/autoscaler/pull/3687

**Comment by tehleet** [comment]
Solved in PR #3687
